careers/ladder.md

```diff
@@ -28,73 +28,161 @@ IC0 > IC1 > IC2 > IC3 > IC4 > IC5
                   M1  > M2  > M3  > M4 > E1 > E2
 ```
 
+### Evaluation criteria
+
+- Knowledge Leadership
+  - Level of knowledge in a specific area
+  - Breadth and depth of technical and soft skills
+- Impact
+  - Nature and impact of problems solved and decisions made
+  - Level of judgment, analysis or creativity required
+- Influence
+  - Nature and level of influence on projects, strategy, and internal and external contacts
+- Discretion
```

Comment:
I think the confusion might be around the 2 different definitions of "discretion." This isn't using its meaning of "keeping secrets," but its meaning of "having the freedom and judgment necessary to make decisions."

It might be more clear if we called this "autonomy" or "independence."